apps/console/src/features/notification-channels/pages/notification-channels.tsx

```diff
@@ -61,17 +73,44 @@ export const NotificationChannelPage: FunctionComponent<NotificationChannelPageI
     const handleEmailSelection = (): void => {
         history.push(AppConstants.getPaths().get("EMAIL_PROVIDER"));
     };
+    
+    /**
+     * Get the page details based on the enabled providers.
+     */
+    const getPageDetails = (): {
+        description: string;
+        pageTitle: string;
+        title: string;
+    } => {
+        if (isEmailProviderEnabled === isSMSProviderEnabled) {
```

Comment:
```suggestion
        if (isEmailProviderEnabled && isSMSProviderEnabled) {
```

Reply:
Need to display this message, when the `email provider` and `sms provider` both are disabled. This is a hypothetical situation and placed as a fallback condition.

Reply:
Can't we go with ```if (!isEmailProviderEnabled && !isSMSProviderEnabled) {```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4697/commits/8bfeed8c422bd4ef5d39051766270113c17c5cc4